README.md

```diff
@@ -2,4 +2,370 @@
 
 [![Join the chat at https://gitter.im/USGS-Astrogeology/isis3_cmake](https://badges.gitter.im/USGS-Astrogeology/isis3_cmake.svg)](https://gitter.im/USGS-Astrogeology/isis3_cmake?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Integrated Software for Imagers and Spectrometers ISIS3 public release site
+## Installation
+
+This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0)+ through conda. If you are a developer, we refer you to our [GitHub repository.](https://github.com/USGS-Astrogeology/ISIS3)
```

Comment:
If you're a developer please go to the exact same page you're on! We should either add a section about compiling ISIS or link to the wiki page about it.